eng/versioning/version_client.txt

```diff
@@ -7,7 +7,7 @@ com.azure:azure-client-sdk-parent;1.7.0;1.7.0
 com.azure:azure-ai-formrecognizer;3.0.0-beta.1;3.0.0-beta.2
 com.azure:azure-ai-textanalytics;5.0.0;5.1.0-beta.1
 com.azure:azure-core;1.7.0;1.8.0-beta.1
-com.azure:azure-core-amqp;1.3.0;1.4.0-beta.1
```

Comment:
This PR is missing this changelog update I think https://github.com/Azure/azure-sdk-for-java/pull/14020/files#diff-eeddfeb67fcf95713246e6f23ff24003R3

Reply:
Yup, that's why the analyze step failed for core. Good catch @samvaity, I missed it.